wagtail/admin/wagtail_hooks.py

```diff
@@ -269,6 +270,18 @@ def register_core_features(features):
         }
     })
 
+    features.register_editor_plugin(
+        'draftail', 'br', draftail_features.BooleanFeature('enableLineBreak')
+    )
```

Comment:
👍 there would also need to be some changes client-side for `enableLineBreak` to be processed [similarly to `enableHorizontalRule`](https://github.com/wagtail/wagtail/blob/6825aab5f764cb9accfd6af47b126e8e0f12f0d2/client/src/components/Draftail/index.js#L66), so its description text is translatable.